services/bettersocial/user/BetterSocialUnblockUserV2.js

```diff
@@ -4,6 +4,17 @@ const {sequelize, User, UserBlockedUser} = require('../../../databases/models');
 const Getstream = require('../../../vendor/getstream');
 const RedisBlockHelper = require('../../redis/helper/RedisBlockHelper');
 const BetterSocialScoreUnblockUser = require('../score/unblock-user');
+
+const isAnonymousUserBlockedFromSignUser = async (selfUserId, anonymousUserId) => {
+  const isBlockedFromSign = await UserBlockedUser.findOne({
+    where: {
+      user_id_blocker: selfUserId,
+      user_id_blocked: anonymousUserId
+    }
+  });
+  if (!isBlockedFromSign) return false;
+  return isBlockedFromSign.post_id ? false : true;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `isAnonymousUserBlockedFromSignUser` checks if a user is blocked from signing in. However, the return statement on line 16 could be simplified for better readability.
```diff
-   if (!isBlockedFromSign) return false;
-   return isBlockedFromSign.post_id ? false : true;
+   return isBlockedFromSign && !isBlockedFromSign.post_id;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->